sys-process/anacron/anacron-2.3-r4.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
```

Comment:
while in theory this change should be safe, please perform it in revbump - great [documentation](https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html) for when you should revbump

So please copy into `-r5` revision, remove stable keywords (`ekeyword ~all sys-process/anacron/anacron-2.3-r5.ebuild`) and push it 😄

Reply:
Done